stations.csv

```diff
@@ -27190,7 +27190,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 29337;Agrigento;agrigento;;;37.31138095340275;13.58164131467822;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;ITAGR;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29338;Bolzano/Bozen;bolzano-bozen;;;46.4961167;11.3546747;;IT;Europe/Rome;t;f;f;f;f;f;;;f;;f;;f;;f;ITBZO;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29339;Messina;messina;;;38.18963375;15.55735599999997;;IT;Europe/Rome;t;f;f;t;f;f;;;f;;f;;f;;f;ITMES;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
-29340;Cerignola;cerignola;;;41.26185045;15.907984650000005;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;ITCER;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+29340;Cerignola;cerignola;;;41.26185045;15.907984650000005;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;ITCER;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```

Comment:
It looks like this one is used for Distribusion, it should not get disabled.

Reply:
Oops, I'll re-enable the station.